web/vtadmin/src/components/dialog/Dialog.tsx

```diff
@@ -121,7 +121,6 @@ const Dialog: React.FC<DialogProps> = ({
                                             className="btn"
                                             onClick={() => {
                                                 onConfirm && onConfirm();
```

Comment:
we could use the new es syntax
```suggestion
                                                onConfirm?.();
```

Reply:
👍